config/initializers/extra_paths.rb

```diff
@@ -0,0 +1,6 @@
+#
+# Add some extra paths here
+Rails.application.configure do
+  config.paths.add File.join('app', 'api'), glob: File.join('**', '*.rb')
+  config.autoload_paths += Dir[Rails.root.join('app', 'api', '*')]
```

Comment:
Not sure, because autoload works with some folders such as controllers/models/etc, but not custom. Well, let me take a look.